src/features/rte/styles/usj-nodes.css

```
@@ -0,0 +1,2707 @@
+.opennote {
+  color: #7777ff;
+}
+
+.usfm table {
+  border-collapse: collapse;
+}
+
+.usfm td {
+  border: 1px solid #000000;
+  page-break-inside: avoid;
+  /* FB27281 adding padding based on font size*/
```

Stylelint (17.14.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)